app/assets/stylesheets/action_page.scss

```diff
@@ -20,64 +36,47 @@
 
 #description {
   margin-top: 3em;
-  h3 {
-    color: #0a639b;
-    line-height: 1.3;
-    * { background-color: #fff; }
-  }
   p, li {
     margin-bottom: 1em;
   }
 }
 
-.tools-wrapper {
-  z-index: 10;
-}
-
 .form-control {
   border-radius: 0;
   padding: 0 0.5em;
   font-size: 0.8em;
 }
 
-#share-buttons-wrapper {
-  position: absolute;
-  text-align: center;
-  right: 20px;
-  top: 0px;
-  display: none;
-  @media screen and (min-width: $md) {
-    position: relative;
-    right: 0;
-    #share-buttons a {
-      display: inline-block !important;
-    }
-  }
-}
-
 #tools {
-
   .tool {
-    background-color: #ffffff;
+    background-color: $gray-dark;
```

Comment:
`#999999` fails WCAG guidelines when white or $off-white is in the foreground, at all text sizes. https://webaim.org/resources/contrastchecker/

We should ask design for a better color here or get creative ourselves. I think it might be hard to address accessibility concerns within the gray-on-gray-on-gray theme.

Reply:
I'll bring this up with Design.

Reply:
Found a very close color that passes (#949494), adding it.

Reply:
Thanks. I'm going to try to start a more general conversation about this with design. Overall the gray theme feels very low-contrast to me.